doc/sources/guide/packaging-osx.rst

```diff
@@ -1,82 +1,192 @@
 Creating packages for OS X
 ==========================
 
-Packaging your application for the OS X 10.6 platform can only be done inside OS X.
-
-The package will only work for the 64 bit OS X. We no longer support 32 bit OS X platforms.
-
 .. note::
-    Currently, packages for OS X can only be generated with Python 2.7. Python 3.3+ support is on the way...
+    - Packaging your application for the OS X platform can only be done inside OS X.
 
-.. _osx_requirements:
```

Comment:
At first I thought it was not being used anymore. But then I found the duplicate https://github.com/kivy/kivy/blob/master/doc/sources/gettingstarted/packaging.rst which I think should either be removed or have the same content as https://github.com/kivy/kivy/blob/master/doc/sources/guide/packaging.rst where the TOC is autogenerated and does not use the label.